website/docs/r/monitor_folder.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
We should remove example from https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/monitor#example-monitor-folder and point user to this new page.